wpeverest/everest-forms
# Fix/mobile container margin issue

### All Submissions:






### Changes proposed in this Pull Request:
![image](https://user-images.githubusercontent.com/66351929/114394919-7cce0c00-9bbb-11eb-9d02-f0cf570e24c6.png)


### How to test the changes in this Pull Request:

1. Checkout to this branch
2. Create a new form 
3. Please check if form is outside of viewport issue still exist or not in mobile device 

### Types of changes:




### Other information:




### Changelog entry

> Fix/mobile container margin issue
